tmail-backend/integration-tests/jmap/jmap-integration-tests-common/src/main/scala/com/linagora/tmail/james/common/PublicAssetSetMethodContract.scala

```diff
@@ -537,6 +537,7 @@ trait PublicAssetSetMethodContract {
            |        "accountId": "$ACCOUNT_ID",
            |        "create": {
            |          "4f29": {
+           |            "blobId": "9c4b1b1b-1992-11ef-b9f4-39749479be62",
```

Comment:
This functional change is unjustified by code relocation.

Reply:
This test checks for the use of invalid properties. It assumes that errors for unknown properties are thrown before errors for missing blob IDs. However, after the made changes this assumption is no longer valid.

Reply:
I do not care about the reason, an integration test change on a code relocation is a big red flag. Behaviour MUST not change.

Remove this line and make it pass.

Reply:
question: In the presence of an invalid json object, does it matter which error is thrown first?

Reply:
I do not care about the reason, an integration test change on a code relocation is a big red flag. Behaviour MUST not change.

Remove this line and make it pass.

Reply:
No logic change = no change in tests

If tests do not pass it means logic test change.

Red flag

What matters is that the test is red.